### PR TITLE
Edit Workers UI View

### PR DESCRIPTION
**Pull request by Timilehin08**
## What type of PR is this? (check all applicable)

- [] Feature
- [] Refactor
- [] Bug Fix
- [] Optimization
- [] Documentation Update
- [x] [] Other (describe: UI Styling)

## Description of what PR does
Completes step 1 of  #14  by styling the view at http://127.0.0.1:3000/workers/:id/edit.
To ensure responsiveness across all screens, especially for the button, col-auto, mx-auto was used.

To build for easier edits in the future, an instance method for the grid layout was used. In the org controller, instances were created and defined for @form_label, @form_text_fields, and @form_submit_button classes. In the edit view, those instances were referenced  Changes to the instances in the controller will be reflected in the view.

## Related PRs and/or Issues (if any)
-
-

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed _(please include reasons for why tests aren't needed)_
- [ ] no, because I need help
- [x] no, they will be added later (please create an Issue for it)

## Added to documentation?

- [ ] Yes, project README
- [ ] No documentation needed

### CODE REVIEW

File: app/controllers/workers_controller.rb [hunk 1]
```diff
@@ -2,12 +2,13 @@ class WorkersController < ApplicationController
   before_action :logged_in_user, only: %i[index show new create edit update]
   before_action :set_worker, only: %i[show edit update]
   rescue_from ActiveRecord::RecordNotFound, with: :render_not_found
-
+  attr_accessor :form_labels, :form_text_fields, :form_submit_button
+
```
Comment (GabrielleEaston) [inline app/controllers/workers_controller.rb:6]: @Timilehin08 Hi Timi! Great work! I wonder what is attr_accessor does? I didn't add it to my code. I wonder If I am missing something important. ☺️

Comment (ncbui) [reply app/controllers/workers_controller.rb:6]: Hey @Timilehin08 , it looks like you're exposing bootstraps classes here but those aren't accessed by controller methods. I'd recommend removing it for this PR.

Comment (ncbui) [reply app/controllers/workers_controller.rb:6]: @GabrielleEaston I've found this[ thread ](https://stackoverflow.com/questions/4370960/what-is-attr-accessor-in-ruby) helped to understand what attr_accessor does. 

TLDR: Ruby doesn't technically have the concept of class variables (even though controller files resemble classes defined in other languages). The attr_accessor method helps to expose variables defined in one class to all the other methods in that file.

**Review by ncbui — CHANGES_REQUESTED** [review]
Great work on this @Timilehin08 ! Styling changes look great but I would recommend removing the changes to the controller.

File: app/controllers/workers_controller.rb (hunk 1 above)
Comment (Timilehin08) [reply app/controllers/workers_controller.rb:6]: Thank you for reviewing! @ncbui @GabrielleEaston.
Oops! Okay, i will.

@GabrielleEaston 
I learnt attr_accessor is a method and class is not a definition in Ruby (it is "just a definition" in languages like Java), but it is an expression that evaluates. It is during this evaluation that the attr_accessor method is invoked which in turn modifies the variables in one method to all other methods in the file. still learning haha, so I definitely used it wrongly here.

Thank you again!, I'll make the changes @ncbui

**Review by ncbui — APPROVED** [review]

**Review by msarit — APPROVED** [review]
Awesome work! 🚀
